ckanext/validation/utils.py

```diff
@@ -1,73 +1,304 @@
 import os
 import logging
+import json
+from io import BytesIO
+from datetime import datetime as dt
 
-from ckan.lib.uploader import ResourceUpload
-from ckantoolkit import config, asbool
+import requests
+from goodtables import validate
+from six import string_types
 
+import ckan.plugins.toolkit as tk
+import ckan.plugins as plugins
+from ckan.model import Session
+from ckan.common import asbool, config
```

Comment:
These should use `toolkit`.

Reply:
\+

Reply:
This will be fixed and pushed by Alex shortly.